plugin/header/README.md

```diff
@@ -0,0 +1,42 @@
+# header
+
+## Name
+
+*header* - modifies the header for all the responses.
+
+## Description
+It ensures that the flags are in the desired state for all the responses. The modifications are made transparently for the client.
+
+## Syntax
+
+~~~
+header {
+    ACTION FLAGS...
+    ACTION FLAGS...
+}
+~~~
+
+* **ACTION**  defines the state for dns flags, allowed values set/clear. 
+* **FLAGS** are the dns flags that will be modified e.g `ra`, `aa` etc.
```

Comment:
should we support _all_ headers? If not, this should list the current supported set